inductiva/structures/post_processing/fenicsx.py

```diff
@@ -0,0 +1,149 @@
+"""Post-process FEniCSx simulation outputs."""
+
+import os
+import tempfile
+
+import pyvista as pv
```

Comment:
pyvista will have to go to an extra package dependency, right @IvanPombo @luispcunha ?

Reply:
Do I need to include this in the `utils/optional_deps.py` file, right? I've already added: 

```
needs_structures_extra_deps = functools.partial(
    _needs_optional_deps,
    ["pyvista"],
    _missing_deps_msg("structures"),
)
```

Since I already have 'pyvista' in optional_deps, can I use it here?